packages/ai-jsx/src/batteries/sidekick/platform/conversation.tsx

```diff
@@ -93,25 +95,38 @@ export function getNextConversationStep(
 
   switch (lastMessage.type) {
     case 'functionCall': {
-      const { name, args } = lastMessage.element.props;
-      const executedFunction = (
-        <ExecuteFunction
-          // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-          func={updatedTools[name]?.func}
-          name={name}
-          args={args}
-        />
-      );
-      // If we are using a tool based on redacted functions, we don't want to redact it further
-      if (tools && !(name in tools)) {
-        return executedFunction;
-      }
-      // Function responses can potentially be very large. In that case, we need
-      // some way of handling that so the context window doesn't blow up.
-      return (
-        <LargeFunctionResponseWrapper numChunks={10} maxLength={10500} failedMaxLength={4000}>
-          {executedFunction}
-        </LargeFunctionResponseWrapper>
+      // Collect all the adjacent function calls and execute them.
```

Comment:
Double check: so now the last message can either be a single `AssistantMessage` or a bunch of `FunctionCall`s?

Reply:
Yup, a single `<ChatCompletion>` can now produce multiple `<FunctionCall>`s.